src/shared/enrollment_op.c

```diff
@@ -540,8 +540,7 @@ static int w_enrollment_concat_src_ip(char *buff, const char* sender_ip, const i
 
     if(sender_ip && !use_src_ip) { // Force an IP
         /* Check if this is strictly an IP address using a regex */
-        if (OS_IsValidIP(sender_ip, NULL))
-        {
+        if (OS_IsValidIP(sender_ip, NULL) == 1) {
```

Comment:
Does not allow CIDR or Netmask?
OS_IsValidIP return 2 when it find CIDR or netmask.

Reply:
Ditto https://github.com/wazuh/wazuh/pull/11268#discussion_r787187266